code/part-one/tests/01-Signing.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const { expect } = require('chai');
```

Comment:
I think we didn't use `chai` for the `part-two` tests. Do we need to use it here?

Reply:
All the tests use chai. The client tests in part-two don't explicitly import it because it gets added globally in the index.html file. The processor tests (also Node based), import it explicitly.